export.py

```diff
@@ -2061,7 +2062,7 @@ def export_object_attributes(ri, scene, ob, visible_objects):
     obj_groups_str = '*'
     for obj_group in scene.renderman.object_groups:
         if ob.name in obj_group.members.keys():
-            obj_groups_str += ',' + obj_group.name
+            obj_groups_str = obj_group.name
```

Comment:
Gah I didn't even think of that. Problem was when that '*' was in the .rib the object group didn't register and I got a black render.  I'll play around with it a bit more.